asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc

```diff
@@ -45,24 +24,23 @@ NOTE: <<ref_tbl_dec_obx18_vmd_mapping>> defines the mapping of the <<acronym_mdi
 
 |OBX-18/EI-1
 |Entity Identifier
-|pm:Mds
-/pm:MetaData+++<wbr/>+++/pm:Udi+++<wbr/>+++/pm:DeviceIdentifier
-|
+|pm:Mds/ext:Extension+++<wbr/>+++/sdpi:EquipmentIdentifier
+|Universal Unique Identifier (UUID) without the prefix *"urn:uuid:"*.

```

Comment:
Currently, the devices unique identifier is a URI, not necessarily limited to UUIDs. I recommend to keep it that way for compatability reasons. Hence, the sdpi:EquipmentIdentifier should contain the URI scheme as well.

Reply:
Didn't we say that we want to make it unified and only support a UUID? The UUID can also be created from an URI.

Reply:
But even then the sdpi:EquipmentIdentifier is a URI after all. The mapping is only possible if poeple used a UUID though. If someone does not leverage UUIDs, it cannot map the data properly. I think that's ok. Otherwise, we need another requirement (if there isn't one already).

``
Maybe rephrase to "Universal Unique Identifier (UUID) URI of scheme **urn:uuid**. If the device does not contain a UUID as equipment identifier, a mapping is not possible.
``

Reply:
HL7 would also support the type "URI". 

My point is that a unified device identifier / device identifier type helps a lot to be interoperable. A URI which is not a UUID can be converted into a UUID. This is not limited to HL7 gateways but also for SDC consumer.

A URI is not specific - it could be any thing a URL, UUID, URN, etc. I was hoping that we could agree on a more specific definition.

@d-gregorczyk: you wrote above "... the devices unique identifier is a URI.." Where is the device URI currently located in BICEPS? Is this just a vendor-specific definition that it has to be a URI?

Reply:
Peter and I came to the conclusion that we require equipment identifiers to be UUIDs. Those can easily be transformed to UUID fields in HL7. If people disagree with that approach, we can discuss an alternative at the SDPi Friday Meetings and/or the formal release reviews.